tests/services/TestMongoService.py

```diff
@@ -40,263 +47,204 @@ def setUp(self):
             "codeName": "NotYetInitialized"
         }
 
-        self.initiate_ok_response = {
-            "ok": 1,
-            "operationTime": 1528365094.1,
-            "$clusterTime": {
-                "clusterTime": 1528365094.1,
-                "signature": {
-                    "hash": "AAAAAAAAAAAAAAAAAAAAAAAAAAA=",
-                    "keyId": 0
-                }
-            }
-        }
+        self.initiate_ok_response = loads("""
```

Comment:
can't we simply put the dict in here then?

Reply:
The 'loads' method of bson.json_util converts $timestamp and other keys like that to something actually json. If that "loads" method changes in the future, the test is going to fail because we're not expecting that change. But that could be exactly what you want.. I don't know?